### PR TITLE
Lamil/enterprise bot

### PR DESCRIPTION
**Pull request by lauren-mills**
Enterprise bot template

### CODE REVIEW

File: csharp_dotnetcore/51.Enterprise-Bot-Template/CogSvcModels/Dispatch/Dispatch.cs [hunk 1]
```diff
@@ -0,0 +1,64 @@
+// <auto-generated>
```
Comment (vishwacsena) [inline csharp_dotnetcore/51.Enterprise-Bot-Template/CogSvcModels/Dispatch/Dispatch.cs]: CogSvcModels should be CongnitiveModels. Also unclear why you have .cs files under cognitiveModels.

File: csharp_dotnetcore/51.Enterprise-Bot-Template/appsettings.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "microsoftAppId": "",
```
Comment (vishwacsena) [inline csharp_dotnetcore/51.Enterprise-Bot-Template/appsettings.json]: This needs to have the botFilePath and botFileSecret

File: csharp_dotnetcore/51.Enterprise-Bot-Template/CogSvcModels/Dispatch/Dispatch.cs (hunk 1 above)
Comment (lauren-mills) [reply csharp_dotnetcore/51.Enterprise-Bot-Template/CogSvcModels/Dispatch/Dispatch.cs]: I wasn't quite sure where the LG files should go once they are created so I put them in CognitiveModels. Thoughts on where they should go?

File: csharp_dotnetcore/51.Enterprise-Bot-Template/appsettings.json (hunk 2 above)
Comment (lauren-mills) [reply csharp_dotnetcore/51.Enterprise-Bot-Template/appsettings.json]: Added.

**Pull request merged**